tests/unit/clients/slack/test_slack_message_builder.py

```diff
@@ -215,15 +215,6 @@ def test_create_compacted_sections_blocks():
     )
 
 
-def test_get_slack_status_icon():
```

Comment:
We removed this test, but maybe we can add a new one that checks that we add the color properly?